### ISSUE TITLE
Add Delta E as part of the design

### ISSUE THREAD

**Issue by pawn002** [pawn002/color-pair-quick-iterator#28]
This should be a secondary metric that never replaces the two main score types of Perceptual Contrast or WCAG.

Perhaps have it as part of a research section below the fold in the smallest viewport supported but visible in above the fold using at typical desktop viewport sizes.

**Comment by pawn002** [comment]
### Color Details Section

Delta E should live in a section of similar granular or extra detail that most folks would not use.

Things like:

* Lightness, Chroma, and Hue
* Saturation
* etc.

Keeping in mind progressive enhancement of the app in larger viewports, **let's try a CSS Grid based approach where color details like the above are under each slider as details specific to each color but at the minimum viewport those details are repositioned in their own discrete section—likely under Options.**

**Comment by pawn002** [comment]
fixed via #44
